brainscore_vision/model_helpers/brain_transformation/__init__.py

```diff
@@ -52,6 +64,27 @@ def __init__(self, identifier,
                                                })
         self.do_behavior = False
 
+    def load_region_layer_map_json(self, identifier):
+        '''
+        Attempts to load the region_layer_map from a JSON file in the model's directory
+        If file exists, load JSON. Otherwise, return None and proceed with legacy layer mapping
+        '''
+        try:
+            importer = import_plugin.ImportPlugin(library_root='brainscore_vision', plugin_type='models', identifier=identifier)
+            model_dir = importer.locate_plugin()
+            region_layer_map_path = os.path.join(os.getcwd(), f'vision/models/{model_dir}', f'region_layer_map/{identifier}.json')
```

Comment:
we're using `pathlib`'s `Path` instead of `os.path` throughout the project

Reply:
I'll fix this